lib/loggingService/writers/file.js

```diff
@@ -200,7 +200,9 @@ FileWriter.prototype.channelFunctionGenerator = function (channelName) {
 		// additional info, formatted (JSON)
 
 		if (entry.data) {
-			msg += '\n' + JSON.stringify(entry.data, null, jsonIndent);
+			for (const dataEntry of entry.data) {
```

Comment:
https://github.com/mage/mage/pull/99/files/9ef0bd6deb77db99c4477af5be2a37d9cb222558#diff-c3caf28db5144d0bfc663414f3eeaf12R250

This should always be an array or undefined.